src/DatabaseSetup.php

```diff
@@ -141,16 +153,18 @@ public function addRemoteAccessUser()
             $password .= $characters[rand(0, strlen($characters) - 1)];
         }
 
-        $sth = $this->dbh->prepare("GRANT ALL ON radius.* TO ?@? IDENTIFIED BY ?");
-        if ($sth->execute([$username, $ipAddress, $password]))
-        {
-            $sth = $this->dbh->prepare("FLUSH PRIVILEGES");
-            $sth->execute();
-            $this->climate->lightMagenta("Added a user with the username $username and the password $password. Copy this username and password, you'll need it!");
-        }
-        else
-        {
-            $this->climate->shout("Failed to create the user!");
+        $stc = $this->dbh->prepare("CREATE USER ?@? IDENTIFIED BY ?");
+        $sth = $this->dbh->prepare("GRANT ALL ON radius.* TO ?@?");
+        for ($i=0; $i < count($ipAddresses); $i++) {
+            if ($stc->execute([$username, $ipAddresses[$i], $password]) && $sth->execute([$username, $ipAddresses[$i]]))
+            {
+                $this->dbh->prepare("FLUSH PRIVILEGES")->execute();
```

Comment:
nit: The flush should be moved out of the loop as it is only really needed once and is not a "light" command for MySQL

Reply:
Thanks.  I fixed that and retested the changes.